style/js/shield_defs.js

```diff
@@ -559,6 +559,52 @@ export function loadShields(shieldImages) {
   shields["US:SC:Business"] = banneredShield(shields["US:SC"], ["BUS"]);
   shields["US:SC:Alternate"] = banneredShield(shields["US:SC"], ["ALT"]);
 
+  shields["US:TX"] = roundedRectShield("white", "black", "black", 1, 1, null);
+  shields["US:TX:Loop"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["LOOP"]
+  );
+  shields["US:TX:Spur"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["SPUR"]
+  );
+  shields["US:TX:Business"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["BUS"]
+  );
+  shields["US:TX:Park"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["PARK"]
+  );
+  shields["US:TX:Beltway"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["BELT"]
+  );
+  shields["US:TX:FM"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["F.M."]
+  );
+  shields["US:TX:RM"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["R.M."]
+  );
+  shields["US:TX:Recreational"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["R"]
+  );
+  shields["US:TX:NASA"] = banneredShield(
+    roundedRectShield("white", "black", "black", 1, 1, null),
+    ["NASA"]
+  );
+
+  shields["US:TX:Toll"] = banneredShield(
+    roundedRectShield("#003f87", "white", "white", 1, 1, null),
+    ["TOLL"]
+  );
```

Comment:
I like how we can distinguish toll express lanes from ordinary lanes:

<img src="https://user-images.githubusercontent.com/1231218/155863189-8cfda27e-3979-4ab6-ba96-998e7c1aaab7.png" width="400" alt="121">

The “TOLL” banner is technically redundant to the blue background, but it does help with usability, since the toll shield very often appears alongside a non-toll shield with the same number.

Do we intend to also add the “TOLL” or “EXPRESS” banner to tolled Interstate and U.S. highways? The express lanes in the Dallas area appear to be tagged with [`network=US:I:Express:Toll`](https://www.openstreetmap.org/relation/8097271) and so on.

Reply:
I think from the informal discussions we've had on Slack, we intend to render toll roads in a different color entirely on the roadway itself.  So I think we probably don't explicitly need to put a toll banner on the highway shields for those roads.

Reply:
The banner guidelines adopted in #186 call for the “TOLL” banner to be kept as in this PR, because toll State Highways aren’t necessarily freeways. For example, [Loop 49](https://en.wikipedia.org/wiki/Texas_State_Highway_Loop_49) is currently an expressway. It’s tagged `US:TX:Loop:Toll`, which is a combination that isn’t handled by this PR yet. On the other hand, `US:I:Express:Toll` may not need both “EXPRESS” and “TOLL” as banners; I’m not sure.

Reply:
Existing `network` values in Texas containing `Toll` are:

* `US:I:Express:Toll`
* `US:TX:Express:Toll`
* `US:TX:Loop:Express:Toll`
* `US:TX:Loop:Toll`

Other banners like "EXPRESS" and "LOOP" should definitely be displayed, and anything with `US:TX:…:Toll` should get a blue shield.

I'm not convinced it's necessary to have the "TOLL" banner if toll roads will have a different color. But for now, it's probably fine to keep it in—we can always revisit this once toll roads get their new style.

Reply:
"EXPR" would be a reasonable abbreviation if we need that in a banner.